ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -1611,8 +1611,8 @@ tests:
     - ref: openshift-extended-test-longduration
     - ref: openshift-e2e-test-qe-report
     workflow: cucushift-installer-rehearse-azure-aks-hypershift-install
-- as: azure-ipi-ovn-hypershift-guest-f14
-  cron: 23 18 6,22 * *
+- as: azure-ipi-ovn-hypershift-guest-f365
```

Comment:
For historical reasons, the other disabled jobs are using f360. Let's use f360 here, too.

Reply:
Updated